core/tests/lighthouse-pages.json

```diff
@@ -0,0 +1,144 @@
+{
+  "shards": {
```

Comment:
If you are doing things by URL, do you still need the predefined shards / does it make more sense to shard dynamically instead, based on which set of tests are run?

Reply:
Hm, I suppose that could be possible, but how would that work in the workflow exactly, would it just be one job and multiple concurrency, don't know if that is most efficient. To dynamically shard we would have to use the lhci directly, or dynamically generate the .lighthouse-1.. files, unless I'm missing a basic approach.

Reply:
I was thinking that we'd just have one file with all the URLs and associated modules defined, and then when analyzing the code for a PR, we'd determine which tests to run and then run them, sharding into multiple workflows if needed (but if e.g. only 3 tests are run across all of lighthouse then we'd just use one shard in total).

But I guess this might be a bit complex, and what you have in terms of two predefined shards (and running a subset o tests in each) seems OK too. So no action needed here really, just wanted to bring the idea up in case you think it might be better.

Reply:
Thanks for clarifying, I might look into that at the workflow PR, though if its too complex I might work on it in the future.

Reply:
@seanlip FYI am going with the approach you highlighted above, since I haven't done the github workflow PR, I have hardcoded the pages for now into the workflow (separated into two shards like before).

Reply:
OK, thanks! Happy to defer to you here.